libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCredentials.h

```diff
@@ -246,7 +246,7 @@ typedef NS_ENUM(NSInteger, SFOAuthCredentialsStorageType){
  @param encrypted Determines if the sensitive data like refreshToken and accessToken should be encrypted
  @return An initialized authentication credential object.
  */
-- (instancetype)initWithIdentifier:(NSString *)theIdentifier clientId:(NSString *)theClientId encrypted:(BOOL)encrypted;
+- (_Nonnull instancetype)initWithIdentifier:( NSString * _Nullable)theIdentifier clientId:( NSString * _Nullable )theClientId encrypted:(BOOL)encrypted;
```

Comment:
init* methods should almost always be `_Nullable`, as their instantiation design pattern allows for returning `nil` values.